src/nss_wrap.cc

```diff
@@ -0,0 +1,560 @@
+#include "nss_wrap.h"
+
+namespace node {
+namespace nss_wrap {
+
+using nss_module::nss_gethostbyname3_r;
+using nss_module::nss_gethostbyname4_r;
+using nss_module::nss_gethostbyaddr2_r;
+using v8::Array;
+using v8::Context;
+using v8::FunctionCallbackInfo;
+using v8::FunctionTemplate;
+using v8::Handle;
+using v8::HandleScope;
+using v8::Integer;
+using v8::Local;
+using v8::Object;
+using v8::String;
+using v8::Value;
+
+NSSReqWrap::NSSReqWrap(Environment* env,
+                       Local<Object> req_wrap_obj,
+                       char* host_val,
+                       int8_t host_type_val)
+    : ReqWrap(env, req_wrap_obj, AsyncWrap::PROVIDER_NSSREQWRAP),
```

Comment:
@mscdex All that needs to happen for that test is that the Handle is created from JS. Doesn't matter if the call fails or not. The test is just showing that it's possible to create the handle from JS.